src/Modules/GraphEngine.ServiceFabric/Trinity.ServiceFabric.GraphEngineService/PackageRoot/Data/trinity_large_memory.xml

```diff
@@ -4,7 +4,8 @@
     <!-- In this demo application we use Trinity.LIKQ as our graph query language. -->
     <Trinity.LIKQ Timeout="10000"/>
     <!-- In this demo application we use Azure blob storage as our persistent storage. -->
-    <Trinity.Azure.Storage ConnectionString="your_connection_string" ContainerName="graphengine"/>
+    <Trinity.Azure.Storage ConnectionString="DefaultEndpointsProtocol=https;AccountName=ikwstorage;AccountKey=kcdkVkeKMjlu+u8mRxnRLOcApQRnFbrXbVQE3j++RPvRdC45FkRThgOnkyaBcn7l3p3nCpICv5D0gzGE9XjAow==;EndpointSuffix=core.windows.net" ContainerName="graphenginestoragespace 
```

Comment:
please remove your key

Reply:
@yatli Yikes - forgot to put the default value back.

Reply:
@yatli I have also upgraded the SF SDK support to 6.4; only a few simple code changes required.